site/hugo.toml

```diff
@@ -122,13 +122,13 @@ github_subdir = "/site/"
 github_branch= "main"
 
 # Google Custom Search Engine ID. Remove or comment out to disable search.
-gcs_engine_id = "d72aa9b2712488cc3"
+# gcs_engine_id = "d72aa9b2712488cc3"
```

Comment:
this just disable search?

Reply:
This disables google search engine, and we use local search engine.